action.yml

```
@@ -70,6 +70,7 @@ runs:
         if [[ "${{ inputs.test }}" =~ false|False ]]; then
           mkdir -p .ssh
           echo -e "-----BEGIN RSA PRIVATE KEY-----\n${{ inputs.wheels-key }}\n-----END RSA PRIVATE KEY-----" >> .ssh/id_rsa
+          ssh-keyscan -l -f .ssh/id_rsa
```

I've not seen this one before, doesn't work on my machines either...

How about:

```suggestion
          ssh-keygen -y -e -f .ssh/id_rsa
```

It will fail or output the public key